test/manual/index.html

```diff
@@ -42,6 +42,11 @@
     <p>Preloaded inlined CSS: Must be green.</p>
     <p><button class="preloaded-button2">Pressing this button</button> displays an alert and should turn red.</p>
   </div>
+  <div class="test lazy-failure-css">
```

Comment:
Could you move this block below `.lazy-css`?

Reply:
Sure, moved it up under `.lazy-css`